### DISCUSSION TITLE
Is there a command tool which like the ollama cli?

### DISCUSSION THREAD

**Discussion by sddzcuigc** [open-webui/open-webui#1057] [General]
I need to set parameters like --vorbose or set parameter num_thread 20

**Comment by jackjiali** [comment]
Hi , May I know if you have fixed the issue? 
For the only 50% CPU core used issue, I can just /set parameter num_thread 20  via ollama command line only within a session, Is it avaliable to set this parameter in the global scope?

**Comment by logichunt319** [comment]
I'm looking for this also. Ollama runs significantly faster when setting the num_thread command in CLI, as per the thread below:
https://github.com/ollama/ollama/issues/2929

Has there been a solution for this yet?

**Comment by logichunt319** [comment]
Hi all - the awesome Open-WebUI dev team have built this into the latest update. Seems like you need to amend it manually for each model.

Inside WebUI, go to "Workspaces" on the top-left, click on the edit button for the model, and in the "Advanced Params" section click "Show", and edit the "num_thread" field with the number of cores.

Thank you dev team - my AI responses are now twice as fast.
